dhctl/pkg/operations/destroy/destroy.go

```diff
@@ -50,53 +66,68 @@ func NewClusterDestroyer(params *Params) *ClusterDestroyer {
 	terraStateLoader := terraform.NewLazyTerraStateLoader(terraform.NewCachedTerraStateLoader(d8Destroyer, state.cache))
 	clusterInfra := infra.NewClusterInfraWithOptions(terraStateLoader, state.cache, infra.ClusterInfraOptions{PhasedExecutionContext: pec})
 
+	staticDestroyer := newStaticMastersDestroyer(params.SSHClient)
```

Comment:
Shouldn't it be a part of DeckhouseDestroyer?

Reply:
we decide that is not important